src/main/webapp/app/core/main/controllers/productController.js

```diff
@@ -32,17 +30,18 @@ app.controller("ProductCtrl", function ($scope, $uibModal, $http, toastr,$locati
     };
 
     productCtrl.getProductPrice = function(product) {
-        var price = 0.0
-        if (product.price) {
+        var price = '';
+        if (product.statusCode == 1) {
```

Comment:
magic number, put it in a variable that describes what this number means. And compare with ===.

Reply:
Eita, esse eu esqueci de mudar, eu criei a  constante pra isso e n coisei